### PR TITLE
Prototype for the activity feed.

### PR DESCRIPTION
**Pull request by aublhr99**
Second try- actually has the jsp and servlet class now.

### CODE REVIEW

**Comment by aublhr99** [comment]
wait I made a change after the pull request and there's a quick error

**Comment by aublhr99** [comment]
Oh wait the error is in my test file. This pull request should be fine.

File: src/main/webapp/WEB-INF/view/activityfeed.jsp [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    <a id="navTitle" href="/">CodeU Chat App</a>
+    <a href="/conversations">Conversations</a>
+      <% if (request.getSession().getAttribute("user") != null) { %>
+    <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+    <% } else { %>
+      <a href="/login">Login</a>
+    <% } %>
+    <a href="/about.jsp">About</a>
+  </nav>
+
```
Comment (zephyllr) [inline src/main/webapp/WEB-INF/view/activityfeed.jsp]: u dont need the nav bar. u can use <jsp:include page="/WEB-INF/includes/header.jsp"/> instead so changes to the nav bar are global

File: src/main/java/codeu/controller/ActivityFeedServlet.java [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+            user.getId(),
+            cleanedMessageContent,
+            Instant.now());
+
+    messageStore.addMessage(message);
+
+    // redirect to a GET request
+    response.sendRedirect("/chat/" + conversationTitle);
+  }
+}
```
Comment (zephyllr) [inline src/main/java/codeu/controller/ActivityFeedServlet.java]: try to keep only the relevant code necessary to run your page. from what i can see, 90% of the code on this page have functionalities related to the chat page, not the activity feed.

**Review by zephyllr — APPROVED** [review]

**Review by zephyllr — APPROVED** [review]

**Pull request merged**